dapp-example/tests/support/bootstrap.js

```diff
@@ -0,0 +1,110 @@
+const HederaSDK = require('@hashgraph/sdk');
+const ethers = require('ethers');
+const hethers = require('@hashgraph/hethers');
+const fs = require('fs');
+const path = require('path');
+
+const client = HederaSDK.Client
+  .forNetwork(JSON.parse(process.env.HEDERA_NETWORK))
+  .setOperator(process.env.OPERATOR_ID_MAIN, process.env.OPERATOR_KEY_MAIN);
+
+const createAccountFromCompressedPublicKey = async function(compressedPublicKey) {
+  const transferTransaction = await (new HederaSDK.TransferTransaction()
+    .addHbarTransfer(HederaSDK.PublicKey.fromString(compressedPublicKey).toAccountId(0, 0), new HederaSDK.Hbar(1000))
+    .addHbarTransfer(HederaSDK.AccountId.fromString(process.env.OPERATOR_ID_MAIN), new HederaSDK.Hbar(-1000))
+    .setNodeAccountIds([HederaSDK.AccountId.fromString('0.0.3')]))
```

Comment:
Q: Is it required to specify the node AccountId?
Won't SDK handle picking this?

Reply:
Yes, it is removed now.